__tests__/app.test.js

```diff
@@ -26,4 +32,43 @@ describe('app', ()=> {
           });     
        });
     });
-});
\ No newline at end of file
+});
+describe('GET /api/articles/:article:id', () => {
+    test.only('status: 200, respond with a single matching article', () => {
+        return request(app)
+        .get(`/api/articles/1`)
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article).toEqual({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 100,
+
+            })
+        })
+    })
```

Comment:
you can rewrite this test in a more programmatic way, to expect specific keys are expected and their values are these strings, this will allow this object to grow without the test failing, as in future you will be asked to add another key here. In the test you have at the moment you are expecting the object to ask for this exact shape and anything else will fail it. See how you are testing the topics for some inspiration - rather than asking for any(String) you can ask for the exact strings to make sure it is the same article.